server/preferences/preferences.route.js

```diff
@@ -3,11 +3,33 @@ const express = require('express');
 const router = express.Router();
 const Preferences = require('./preferences');
 
-router.get('/preferences', async (req, res) => {
-  res.json(await Preferences.fetch(req.body));
-});
-router.put('/preferences', async (req, res) => {
-  res.json(await Preferences.update(req.body));
-});
+/**
+ * Routes for retrieving, creating or updating an entity's preferences
+ */
+router.use('/preferences', [
+  router.get('/', async (req, res, next) => {
+    try {
+      res.json(await Preferences.get({ email: req.user.email }));
+    } catch (error) {
+      next(error);
+    }
+  }),
+  router.put('/', async (req, res, next) => {
+    const preferences = req.body;
```

Comment:
Do you really need this `const`? Is it just to allow one use of es6 shorthand? I guess you're adding the characters either way...

Reply:
No reason other to allow for shorthand, but I should have also declared email as well to be consistent.

Reply:
I updated this so that email is also using shorthand for consistency.